app/views/teacher_interface/registration_number/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :page_title, "Registration number" %>
+<% content_for :back_link_url, teacher_interface_application_form_path(@application_form) %>
+
+<%= form_with model: @registration_number_form, url: [:teacher_interface, @application_form, :registration_number], method: :put do |f| %>
+  <%= f.govuk_text_field :registration_number,
+                         label: { text: "What is your registration number?", size: "l", tag: "h1" },
```

Comment:
Indenting is off 'ere 😁

Reply:
Oh that's actually how I've been indenting in the views, it seems like we're a bit inconsistent in the app but I've seen multiple ways of doing it!